app/scripts/models/reliers/oauth.js

```diff
@@ -95,6 +97,23 @@ define([
       return !! this.getSearchParam('code');
     },
 
+    /**
+     * Sets relier properties from the resume token value
+     * @private
+     */
+    _parseResumeToken: function () {
+      var resumeToken = this.getSearchParam('resume');
+      var parsedResumeToken = ResumeToken.parse(resumeToken);
+
+      if (parsedResumeToken) {
+        _.each(RELIER_FIELDS_IN_RESUME_TOKEN, function (itemName) {
+          if (parsedResumeToken.hasOwnProperty(itemName)) {
```

Comment:
A more robust way is to use `Object.prototype.hasOwnProperty.call(object, key)` rather than `object.hasOwnProperty(key)`.